models/tokens/ethereum/tokens_ethereum_transfers.sql

```diff
@@ -0,0 +1,15 @@
+{{ config(
+        alias = 'transfers',
+        materialized = 'view',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
should we remove this post hook for now? until everything is final? then maybe only expose the cross-chain transfers (if that is the final ultimate plan)?

Reply:
Not sure if we should only do cross chain here because of the size but will still remove it until we are fully ready.